frontend/rollup.config.js

```diff
@@ -52,5 +52,8 @@ export default [{
         copy({
             targets: [{ src: '../thumbnail.png', dest: 'dist/listing' }]
         }),
+        copy({
+            targets: [{ src: '../manifest.json', dest: 'dist/manifest.json' }]
```

Comment:
Paths should be `src: 'manifest.json', dest: 'dist/viewer'`:
  - The path of the `src` is relative to `frontend` and the manifest is indeed in that directory, not at the root of the repo
  - It is reads from the viewer code base (both when serving with embedded data & using the dev mode)

Reply:
Err, whoops. Fixed and ran `rollup` to confirm.